docs/howto/upgrade-version.txt

```diff
@@ -69,9 +94,9 @@ Testing
 =======
 
 When the new environment is set up, :doc:`run the full test suite
-</topics/testing/overview>` for your application. In Python 2.7+, deprecation
-warnings are silenced by default. It is useful to turn the warnings on so they
-are shown in the test output (you can also use the flag if you test your app
+</topics/testing/overview>` for your application. Again, it is useful to turn
```

Comment:
it's
turn on deprecation warnings
(also flow the "(you can.." bits to the previous line to keep consistent line breaks)